### PR TITLE
Minor test improvements

### PR DESCRIPTION
**Pull request by aweng98**
- latest sbt version
- test changes in workbench-lib PR (chrome options). https://github.com/broadinstitute/workbench-libs/pull/155

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by aweng98** [comment]
jenkins retest

File: automation/build.sbt [hunk 1]
```diff
@@ -58,7 +58,7 @@ Test / logBuffered := false
   *  Warning: can't set too high (set at 10 would crashes OS)
   *  This is not number of threads in each JVM. That would be up to sbt.
   */
-Global / concurrentRestrictions := Seq(Tags.limit(Tags.ForkedTestGroup, 5))
+Global / concurrentRestrictions := Seq(Tags.limit(Tags.ForkedTestGroup, 4))
```
Comment (aweng98) [inline automation/build.sbt]: lower # of JVM to reduce parallelism

**Comment by aweng98** [comment]
jenkins retest

File: automation/src/test/scala/GoogleSpec.scala [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import org.broadinstitute.dsde.workbench.service.test.WebBrowserSpec
-import org.scalatest.{FlatSpec, Matchers, ParallelTestExecution}
+import org.scalatest.{FlatSpec, Ignore, Matchers, ParallelTestExecution}
 import org.scalatest.selenium.WebBrowser
 
 /**
   * Example test for running with Selenium Grid until we work out docker
   * networking to be able to access FiaBitC.
   */
+@Ignore
```
Comment (aweng98) [inline automation/src/test/scala/GoogleSpec.scala]: ignore `GoogleSpec` when run tests on `fiab` `dev` `alpha` `staging` env

File: automation/project/Dependencies.scala [hunk 3]
```diff
@@ -4,7 +4,7 @@ import sbt.Keys._
 object Dependencies {
   val jacksonV = "2.9.0"
 
-  val serviceTestV = "0.12-da66918"
+  val serviceTestV = "0.12-283ba96"
```
Comment (aweng98) [inline automation/project/Dependencies.scala]: latest workbench-lib

File: automation/project/build.properties [hunk 4]
```diff
@@ -1 +1 @@
-sbt.version = 1.1.4
\ No newline at end of file
+sbt.version = 1.2.1
```
Comment (aweng98) [inline automation/project/build.properties]: latest sbt release

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala [hunk 5]
```diff
@@ -144,6 +144,8 @@ class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodCo
     val suggestionTextfield: Element = find(testId(dataTestId)).get
     val expandedDropdownId = suggestionTextfield.underlying.getAttribute("aria-owns")
 
+    click on find(testId("test.hello.response-text-input")).get // force page scrolls down
```
Comment (aweng98) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala]: While testing PR, ran into a flaky test UI issue, this handles it.

File: automation/src/test/scala/GoogleSpec.scala (hunk 2 above)
Comment (breilly2) [reply automation/src/test/scala/GoogleSpec.scala]: I think this whole test can go away. It was created as a first test of our selenium infrastructure. It doesn't even touch FireCloud.

Comment (aweng98) [reply automation/src/test/scala/GoogleSpec.scala]: `GoogleSpec` is gone.

**Comment by aweng98** [comment]
jenkins retest

**Review by gdlex4015 — APPROVED** [review]
Once tests pass you got my blessing

**Pull request closed without merging**